packages/berry-pnpify/tests/PnPRootLocator.test.ts

```diff
@@ -0,0 +1,66 @@
+import { PnPRootLocator } from '../sources/PnPRootLocator';
+
+describe('PnPRootLocator should', () => {
+  let locator;
+  let checkCount;
+  beforeEach(() => {
+    checkCount = 0;
+    locator = new PnPRootLocator({
+      existsSync: (pathname) => {
+        checkCount++;
```

Comment:
https://jestjs.io/docs/en/jest-object#jestfnimplementation

Reply:
Good point, thank you! Switched to using `jest.fn` as per your comment